starters/deno-oak-denodb/src/rest/services/healtcheck_service.ts

```diff
@@ -0,0 +1,39 @@
+import { cache } from '../../cache/mod.ts';
+import { logger } from '../../util/logger.ts';
+import { db } from '../../db/db.ts';
+import { PostgresConnector } from '../../../deps.ts';
+
+export async function isCacheRunning(): Promise<boolean> {
+	let isRunning;
+
+	try {
+		isRunning = await cache.pingRedis() === 'PONG';
+	} catch (e) {
+		isRunning = false;
+		logger.error(e);
+	}
+
+	return isRunning;
+}
+
+export async function isDatabaseRunning(): Promise<boolean> {
+	let isRunning;
+
+	try {
+		const connector = db.getConnector() as PostgresConnector;
+		// We're accessing a private property here, but this is to work around a bug where the connector ping()
+		// method is not implemented properly (below is the proper implementation)
+		// This should be fixed in this PR: https://github.com/eveningkid/denodb/pull/363
+		const client = connector._client;
+		const [firstRow] = (
+			await client.queryObject<{ result: number }>('SELECT 1 + 1 as result')
```

Comment:
I went with `SELECT version()` so in the health check it provides information about the database in the node kit. This seems to be okay to me as well, just a note.